api/envoy/api/v2/eds.proto

```diff
@@ -50,12 +50,21 @@ message ClusterLoadAssignment {
 
   // Load balancing policy settings.
   message Policy {
-    // Percentage of traffic (0-100) that should be dropped. This
-    // action allows protection of upstream hosts should they unable to
-    // recover from an outage or should they be unable to autoscale and hence
-    // overall incoming traffic volume need to be trimmed to protect them.
-    // [#v2-api-diff: This is known as maintenance mode in v1.]
-    double drop_overload = 1 [(validate.rules).double = {gte: 0, lte: 100}];
```

Comment:
I gather this was never implemented, but do we need to keep this in place (but marked deprecated) until 1.8.0 on the off chance that someone is setting it?